src/DynamoApplications/StartupUtils.cs

```diff
@@ -63,6 +63,8 @@ internal class CMDLineOptions
         public string SessionId { get; set; } = String.Empty;
         [Option("CERLocation", Required = false, HelpText = "Specify the crash error report tool location on disk.")]
         public string CERLocation { get; set; } = String.Empty;
+        [Option("ServiceMode", Required = false, HelpText = "Specify the service mode startup.")]
+        public bool ServiceMode { get; set; }
```

Comment:
I guess I missed this, without the helptext, do we not display anything right now?